<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 728, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 834, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 894, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_syntax.py", line 41, in check_recipe
    spec = cast(str, ro_parser.get_value(path))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 983, in get_value
    raise KeyError(f"No value/key found at path {path!r}")
KeyError: "No value/key found at path '/requirements/host/0'"

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_f5vmlxbzj2/clone` failed. (See above for error)
</details>

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -1,15 +1,12 @@
 #!/bin/bash
 
-if [[ "${target_platform}" == "osx-arm64" ]]; then
-  go build -v -o $PREFIX/bin/go-licenses
-
-  # Build for osx-64 as this can also be run in cross-compilation mode.
-  GOARCH=amd64 go build -v -o go-licenses-native
-  ./go-licenses-native save . --save_path=./license-files
+if [[ "${target_platform}" == "win-64" ]]; then
+  go build -v -o $PREFIX/bin/go-licenses.exe
 else
-  go install -v github.com/google/go-licenses
-  go-licenses save . --save_path=./license-files
+  go build -v -o $PREFIX/bin/go-licenses
 fi
+
+go-licenses save . --save_path=./license-files
 
 # TODO: remove if not actually needed, see #6
 # rm -r ./license-files/github.com/google/licenseclassifier/licenses

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -5,29 +5,37 @@ package:
   version: {{ version }}
 
 source:
-  - url: https://github.com/google/go-licenses/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 70c1ceb7c342ceb79b63a76caafb13ea3796a51715c742a482eb9d85277311e7
+  url: https://github.com/google/go-licenses/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 70c1ceb7c342ceb79b63a76caafb13ea3796a51715c742a482eb9d85277311e7
 
 build:
   number: 0
+  skip: true  # [s390x]
 
 requirements:
   build:
     - {{ compiler('go') }}
-    - posix  # [win]
   host:
   run:
     - go
 
 test:
+  requires:
+  source_files:
+    - LICENSE
+    - check.go
+    - e2e_test.go
+    - go.mod
+    - go.sum
+    - main.go
+    - report.go
+    - internal
+    - licenses
+    - testdata
   commands:
-    - git clone --depth=1 "https://github.com/google/go-licenses"
-    - cd go-licenses
     - go-licenses csv . | findstr "Apache-2.0 BSD-.-Clause MIT"             # [win]
     - go-licenses csv . | grep 'Apache-2\.0\|BSD-.-Clause\|MIT\|Unlicense'  # [not win]
-  requires:
-    - git
-    - go
+    - go test -v .                                                          # [not win]
 
 about:
   home: https://github.com/google/go-licenses
@@ -37,6 +45,12 @@ about:
     - LICENSE
     - license-files/
   summary: A tool to collect licenses from the dependency tree of a Go package in order to comply with redistribution terms.
+  description: |
+    go-licenses analyzes the dependency tree of a Go package/binary. It can output a report on the libraries used
+    and under what license they can be used. It can also collect all of the license documents, copyright notices and
+    source code into a directory in order to comply with license terms on redistribution.
+  doc_url: https://github.com/google/go-licenses/blob/master/README.md
+  dev_url: https://github.com/google/go-licenses
 
 extra:
   recipe-maintainers:

